Makefile

```diff
@@ -76,7 +76,7 @@ ifeq ($(TAG_LATEST), true)
 endif
 
 test:
-	@go test -race -cover -coverprofile=coverage.out -covermode=atomic ./cmd/... ./pkg/...
+	@go test -cover -coverprofile=coverage.out -covermode=atomic ./cmd/... ./pkg/...
```

Comment:
can we have the -race option as well, assuming its to find race among go routines?

Reply:
ok.